index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <main class="container">
+        <header>
+            <div class="logo">logos</div>
+            <div class="humbuger"><img src="images/icones/Union.png" alt="humburger"></div>
+        </header>
+        <div class="contain">
+            <section class="container-title">
+                <h1 class="item1">Faustin Makutano</h1>
+                <h2 class="item2">Software Developer</h2>
+                <div class="literature">
+                    <p class="item3">I can help you build a product , feature or website Look through some of my work and experience!<br> If you like what you see and have a project you need coded,<br> don’t hestiate to contact me.</p>
+                </div>
+            </section>
+        </div>
+        <section class="border-bottom">
+            <ul class="skills-list">
+                <li class="language">
+                    <h3 class="skill-title">Language</h3>
+                    <ul class="sublist">
+                        <li class="skill">JavaScript</li>
+                        <li class="skill">Ruby</li>
+                        <li class="skill">Html</li>
+                        <li class="skill">Css</li>
+                    </ul>
+                </li>
+                <li class="frameworks">
+                    <h3 class="skill-title">Frameworks</h3>
+                    <ul class="sublist">
+                        <li class="skill">Bootstrap</li>
+                        <li class="skill">Ruby on Rails</li>
+                        <li class="skill">RSPec</li>
+                        <li class="skill">Capy Bary</li>
+                        <li class="skill">Selenium</li>
+                    </ul>
+                </li>
+                <li class="skills">
+                    <h3 class="skill-title">Skills</h3>
+                    <ul class="sublist">
+                        <li class="skill">Codekit</li>
+                        <li class="skill">Github</li>
+                        <li class="skill">Codpen</li>
+                        <li class="skill">Gitlab</li>
+                        <li class="skill">Terminal</li>
+                    </ul>
+                </li>
+            </ul>
+        </section>
+        <hr/>
+        <div class="icones">
+            <img src="images/icones/sector.png" alt="icon1 ">
+            <img src="images/icones/Vecto.png" alt=" icon2">
+            <img src="images/icones/Vector.png" alt="icon3 ">
+            <img src="images/icones/Vector.svg" alt="icon4 ">
+        </div>
```

Comment:
- [ ] Kindly make sure your icons use a `ul` tag to group them, and li per icon, and each icon has a link tag.

Reply:
- [ ] Kindly check on your design of the icons there is a need for spacing between the icons, also I cant see your arrow, below is a comparison between your design and the Figma design:


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/170674446-5e202884-6deb-4b3a-a6c7-e2e38405cfbf.png) |  ![image](https://user-images.githubusercontent.com/61408860/170674506-ab6c2c39-db56-4892-b1f4-d56bba98d361.png)